golang/Dockerfile.template

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.18.9 as base
+FROM golang:${GOLANG_VERSION} as base
```

Comment:
You could use an ARG, avoids using a script :)

Reply:
the problem with these methods is that dependabot won't be able to create the pull requests to update the dockerfiles.